### ISSUE TITLE
Ideas for Roadmap

### ISSUE THREAD

**Issue by Rbn3D** [Rbn3D/GTAV_NativeDB_Explorer#2]
Please note that this is just a set of ideas that may (or may not) get implemented on the future. 

* Generate natives.h
     * Description as comment? (optional)
* Parser can read natives from natives.json
* Editing capabilities?

**Comment by peaches6** [comment]
Good idea, but sadly NativeDB doesn't have as many parameter descriptions as oppose to other sources, also the fact that NativeDB is free for anyone to edit doesn't help.
